frontend/src/pages/procurement-library/components/CaseStudy.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { ClockIcon } from "../../../lib/icon";
+
+const CaseStudy = ({ study }) => {
+  const formatDate = (dateString) => {
+    const options = { year: "numeric", month: "long", day: "numeric" };
+    return new Date(dateString).toLocaleDateString("undefined", options);
```

Comment:
Hi mas @ifirmawan , 
There's an error on this line of code.
<img width="1211" alt="Screenshot 2025-03-07 at 6 43 40 PM" src="https://github.com/user-attachments/assets/2831c526-e552-4d9d-9afc-b0406b154f93" />